NetSDK/Domain/KeyImpression.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Splitio.Domain
+{
+    public class KeyImpression
+    {
+        public string keyName { get; set; }
+        public string treatment { get; set; }
+        public long time { get; set; }
+        //public long changeNumber { get; set; }
```

Comment:
I added it and then removed it because I had the feature name in the main structure as index and use it from that structure when I send bulk impressions to api